### PR TITLE
Refactor CI

### PR DESCRIPTION
**Pull request by TiagoCavalcante**
It is better to use npm's script `test` in CI because if it is updated CI is too.

Also, after thinking, I reached the conclusion that http-server is a dependency, not a developement dependency.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,9 +27,11 @@
 		"start-server": "http-server &",
 		"test": "npm run start-server && backstop test"
 	},
+	"dependencies": {
+		"http-server": "^13"
```
Comment (rocky) [inline package.json:31]: I am not sure that http-server is  a dependency.  There are other http servers that will work as well. 

And the code doesn't care if there is a http server running.  It is only if you want to demo this that this is useful.

Comment (TiagoCavalcante) [reply package.json:31]: > I am not sure that http-server is a dependency. There are other http servers that will work as well.

Yes, not just others, but any other.

> And the code doesn't care if there is a http server running. It is only if you want to demo this that this is useful.

And that is the point: if we let it be a dependency, in README the process to see something working would be easier, would be the default: install dependencies and run a script. And if that install the development dependencies unnecessary things would be installed.

**Comment by TiagoCavalcante** [comment]
The results were very weird.

Before:
```
10000 iterations of ElementData...
  2.436314 secs for: ElementData["Gold", "AtomicNumber"]     

10000 iterations of NextPrime...
  3.416719 secs for: NextPrime[1, 1]
```

After:
```
10000 iterations of ElementData...
  2.569624 secs for: ElementData["Gold", "AtomicNumber"]

10000 iterations of NextPrime...
  3.259799 secs for: NextPrime[1, 1]
```

File: package.json (hunk 1 above)
Comment (rocky) [reply package.json:31]: Ok. I see your point. If this turns out not quite right in the future it can be changed. But, sure, let's go with this.

**Pull request merged**